packages/eslint-config-kununu/index.js

```diff
@@ -1,18 +1,13 @@
 module.exports = {
-  extends: 'airbnb', // Many strict rules for ECMAScript and React
-
-  parser: 'babel-eslint',
```

Comment:
This was removed because it's the solution to this issue https://github.com/babel/babel-eslint/issues/530#issuecomment-385774262